src/System.Private.Uri/src/System/IPv4AddressHelper.cs

```diff
@@ -184,7 +184,7 @@ internal static unsafe long ParseNonCanonical(char* name, int start, ref int end
         {
             int numberBase = Decimal;
             char ch;
-            long[] parts = new long[4];
+            Span<long> parts = stackalloc long[4];
```

Comment:
Should we make a similar change in https://github.com/dotnet/corefx/blob/1fc5e1e02804665a5b9755b665122e9811276702/src/System.Net.Primitives/src/System/Net/IPv4AddressHelper.cs#L168?

It looks like there are 2 very similar versions of this file.

Reply:
> Should we make a similar change in

That ones already not allocating.

> It looks like there are 2 very similar versions of this file.

We should definitely consolidate them.  Just not as part of this change :)